--- a/experiment_planning/experiment_planners/default_experiment_planner.py
+++ b/experiment_planning/experiment_planners/default_experiment_planner.py
@@ -458,7 +458,7 @@
 
         # instead of writing all that into the plans we just copy the original file. More files, but less crowded
         # per file.
-        shutil.copy(join(self.raw_dataset_folder, 'dataset.json'),
+        shutil.copyfile(join(self.raw_dataset_folder, 'dataset.json'),
                     join(nnUNet_preprocessed, self.dataset_name, 'dataset.json'))
 
         # json is stupid and I hate it... "Object of type int64 is not JSON serializable" -> my ass

--- a/experiment_planning/plan_and_preprocess_api.py
+++ b/experiment_planning/plan_and_preprocess_api.py
@@ -114,7 +114,7 @@
         preprocessor = configuration_manager.preprocessor_class(verbose=verbose)
         preprocessor.run(dataset_id, c, plans_identifier, num_processes=n)
     maybe_mkdir_p(join(nnUNet_preprocessed, dataset_name, 'gt_segmentations'))
-    [shutil.copy(i, join(join(nnUNet_preprocessed, dataset_name, 'gt_segmentations'))) for i in
+    [shutil.copyfile(i, join(join(nnUNet_preprocessed, dataset_name, 'gt_segmentations'), i.split('/')[-1])) for i in
      subfiles(join(nnUNet_raw, dataset_name, 'labelsTr'))]
